### PR TITLE
docs(v2): Add tinaeldevresse.eu to Docusaurus' gallery

### PR DESCRIPTION
**Pull request by HunteRoi**
## Motivation

Just willing to add my personal blog to your gallery 😄 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes!

## Test Plan

N/A

## Related PRs

N/A

### CODE REVIEW

File: website/src/data/users.js [hunk 1]
```diff
@@ -998,6 +998,14 @@ const Users = [
     source: null,
     tags: [ 'facebook'],
   },
+  {
+    title: 'Tinaël Devresse',
+    description: 'Personal website of Tinaël Devresse, a blog about anything that cross my mind which I think could interest y\'all.',
+    preview: require('./showcase/tinaeldevresse.png'),
+    website: 'https://tinaeldevress.eu',
```
Comment (slorber) [inline website/src/data/users.js:1005]:
```suggestion
    website: 'https://tinaeldevresse.eu/',
```

hopefully I double check everything 😅

File: website/src/data/users.js [hunk 2]
```diff
@@ -998,6 +998,14 @@ const Users = [
...
+    source: 'https://github.com/hunteroi/tinaeldevresse.eu',
+    tags: ['personal', 'opensource', 'i18n']
```
Comment (slorber) [inline website/src/data/users.js:1007]:
```suggestion
    tags: ['personal', 'opensource']
```

Does not seem to use i18n support for now

**Review by slorber — APPROVED** [review]
Thanks, just changing a few details

**Comment by HunteRoi** [comment]
Just saw you removed the i18n tag although I am currently translating everything. Would you mind if I add it back once I have pushed the i18n feature from v2.0.0-beta.0 ? @slorber

**Comment by slorber** [comment]
sure no problem, just don't want to have tags that are not yet visible in your site

**Pull request merged**